contracts/StorageContract.sol

```diff
@@ -145,7 +148,9 @@ abstract contract StorageContract is DecentralizedKV, ReentrancyGuardTransient {
     }
 
     /// @notice People can sent ETH to the contract.
-    function sendValue() public payable {}
+    function sendValue() public payable {
```

Comment:
Does the value payed by deployment like the following need to be added to accPrepaidAmount?
await EthStorageUpgradeableProxy.deploy(impl, ownerAddress, data, {
    gasPrice: gasPrice,
    value: prepaidAmount
});

Reply:
I guess we should use sendValue in this case.

Reply:
https://github.com/ethstorage/storage-contracts-v1/pull/86/files 
this may be in conflict with this change.